app/jobs/parse_report_job.rb

```diff
@@ -6,7 +6,7 @@ def perform(file, account, b64_identity)
     parser = XCCDFReportParser.new(file, account, b64_identity)
     parser.save_rule_results
   ensure
-    File.delete(file)
+    File.delete(parser.report_path) if File.exist? parser.report_path
```

Comment:
This shouldn't really get called as `Tempfile` destroys the file as soon as it's not reclaimed by the context, at garbage collection time, which is why I'm ensuring the file exists before removing it